app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,24 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_tag url: '/login', layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-4" do |f|
+        %form.sign_in
```

Comment:
You are using `%form` html tag here - hard to believe this form works. You can not nest 2 forms.